Resources/Textures/Objects/Fun/Spai.rsi/meta.json

```diff
@@ -0,0 +1,35 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from tgstation at commit https://github.com/tgstation/tgstation/commit/9ddb8cf084e292571d4e9c79745db25befbd82fe. pai-searching-overlay heavily modified.",
+  "size": {
+    "x": 32,
+    "y": 32
+  },
+  "states": [
+    {
+      "name": "Spai-base"
```

Comment:
I also can't get why would you just copy paste the base PAI sprite. Just make custom Synciate PAI overlays and apply the base that is already a thing